boards/arm/mr_canhubk3/Kconfig.defconfig

```diff
@@ -6,4 +6,18 @@ if BOARD_MR_CANHUBK3
 config BOARD
 	default "mr_canhubk3"
 
+if SERIAL
+
+config UART_CONSOLE
+	default y
+
+endif # SERIAL
+
+if SHELL
+
+config SHELL_STACK_SIZE
```

Comment:
Non blocking- why does this board need to change the default shell stack size?

Reply:
Removed. This is not needed currently, it was due to using another UART driver at the time this was developed